api-report/container-runtime.api.md

```diff
@@ -680,9 +688,10 @@ export type SubmitSummaryResult = IBaseSummarizeResult | IGenerateSummaryTreeRes
 
 // @public
 export class Summarizer extends EventEmitter implements ISummarizer {
+    // Warning: (ae-forgotten-export) The symbol "IConnectableRuntime2" needs to be exported by the entry point index.d.ts
```

Comment:
Ugh... I was trying to avoid this very warning.  Taking a look...

Reply:
Looks like there's precedence for this kind of thing, see the constructor of ContainerRuntime, it has the same warnings.  Would be great to avoid export classes wherever possible, but we need them for e2e tests I guess.

Reply:
So the question is - if we're going to bend the rules is it better to relax requirements around breaking "internal" interfaces which are part of the exported API, or to just remove those interfaces but have the warnings, which means we can change them without notice which could break someone just the same.

Answer: let's strive to not bend the rules :P  Having internal markup like @CraigMacomber has talked about sounds so nice.

For now, I'm going to revert most of this PR, and simply add back the thing that was removed so this interface isn't broken